src/github.com/getlantern/lantern/lantern.go

```diff
@@ -127,6 +130,7 @@ func run(configDir string) {
 		func(cfg *config.Config) {},                   // onConfigUpdate
 		&userConfig{},
 		func(err error) {}, // onError
+		base64.StdEncoding.EncodeToString(uuid.NodeID()),
```

Comment:
Nice! That's actually a lot cleaner. Do we also need to update ``GetToken()`` to return this?